### ISSUE TITLE
Could not find executable '...ampersand_filter.sh'

### ISSUE THREAD

**Issue by ericpgreen** [crsh/papaja#162]
I installed `papaja` on a Ubuntu 16.04.3 LTS machine running R version 3.4.2 (2017-09-28). I get the following error when I try to knit the default template. I do not get this error on my local machine.

```
pandoc: Error running filter /home/guest/R/x86_64-pc-linux-gnu-library/3.4/papaja/rmarkdown/templates/apa6/resources/ampersand_filter.sh:
Could not find executable '/home/guest/R/x86_64-pc-linux-gnu-library/3.4/papaja/rmarkdown/templates/apa6/resources/ampersand_filter.sh'.
Error: pandoc document conversion failed with error 83
Execution halted
```

Do you have any guesses about how to fix the problem?

**Comment by ericpgreen** [comment]
A colleague took a look for me and said while we were missing some dependencies, there is another problem: `ampersand_filter.sh` is not executable. I can fix that on my end, but I'd like to show the package to students who are new to RStudio. They don't know enough to work in the terminal. When you have time, could you fix the problem with missing executable flags? It's a great template, and I'm excited to share it with my students.

**Comment by crsh** [comment]
Hi Eric,

thanks for bringing this to my attention. I'm not sure where there problem lies. I work on 16.04 with the current R version myself and never had any problems. The executable flag should be set upon installation. Could this possibly be related to using the Linux guest account? Could you try on the same machine with another user account?

Also, are your students working on Linux computers, too? I have never had any issues with the executability flag on any other system.

**Comment by ericpgreen** [comment]
Hi,

We use RStudio server with Docker at the university, so all accounts are guest. Maybe that is the issue.

E

**Comment by crsh** [comment]
Thanks for the quick response. I'm on vacation right now. When I'm back at the office, I'll try installing and using `papaja` on our RStudio server and see if I can reproduce the problem.

**Comment by ericpgreen** [comment]
Awesome. Thanks.

**Comment by mariusbarth** [comment]
Hi Eric,

on our our RStudio Server, I can observe a similar problem if I install `papaja` as a user into my user library. However, the problem disappears if I install `papaja` to the system library with super-user rights. If packages in the system library are available to all users (including guests) on your server, this might be a fix for your problem.
I use the following R code to install to my system library:
```
devtools::with_libpaths(new = "/usr/lib/R/library/", install_github('crsh/papaja'))
```
I would be glaf to hear from you if this did the job.

**Comment by ericpgreen** [comment]
Hi,

Yes, that did the trick.

Thanks!

**Comment by mariusbarth** [comment]
Hi Eric,

that sounds great! Thanks for testing and have fun with `papaja`!

**Comment by mariusbarth** [comment]
Hi! The admin has to allow users to execute the bash file, which is done via

`sudo chmod 755 ampersand_filter.sh`

on ubuntu. Just ask your admin. I will try to add a short tutorial on this in our manual, soon.

**Comment by mariusbarth** [comment]
Yes, that should fix the issue. I will try to add a short section on this in our manual, soon.

**Comment by jorgesinval** [comment]
Thanks, I'm trying to produce a paper with it. It would be good to have an automatic function to report (APA way) the GoFs from a lavaan object (`fitmeasures`).

**Comment by crsh** [comment]
Hi Jorge, absolutely. It's on the list of things to do for a future release. Join the conversation [here](https://github.com/crsh/papaja/issues/181).

**Comment by crsh** [comment]
One other note: Once RStudio ships with `pandoc` 2.0 we will be able to apply the ampersand filter using a native R script. This means that the error you are encountering will be resolved by moving to `pandoc` 2.0. I'm, however, not sure when this will be the case.

**Comment by jorgesinval** [comment]
Thanks! I will check the discussion about lavaan's objects.
